web/src/components/Testimonial/Testimonial.tsx

```diff
@@ -28,7 +28,6 @@ export const Testimonial: React.FC<TestimonialProps> = (
 		quoteText,
 		quoteName,
 		quoteRole,
-		className,
```

Comment:
We need to keep this otherwise we don't get any classes on the wrapping testimonial div (you can see this changing in the snapshot files)